src/helpers/EmailNotificationAPI.js

```diff
@@ -0,0 +1,66 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+
+/**
+ * Represents a EmailNotificationAPI.
+ */
+class EmailNotificationAPI {
+  /**
+   * @constructor
+   * @param {Object} emailPayload - The title of the book.
+   */
+  constructor(emailPayload) {
+    this.mailOptions = {
+      from: `"Merry Ah 👻" <${process.env.EMAILUSER}>`, // sender address
+      to: emailPayload.recipient, // list of receivers
+      subject: emailPayload.subject, // Subject line
+      text: emailPayload.message, // plain text body
+      html: `<p> ${emailPayload.message} </p>` // html body
+    };
+  }
+
+
+  /**
+   * @Represents a transportCreator
+   * @return {Object|string} nodemailer Transport OR Error message string.
+   */
+  static transportCreator() {
+    const {
+      EMAILHOST, EMAILPORT, EMAILUSER, EMAILPASS
+    } = process.env;
+    if (!EMAILUSER || !EMAILPASS || !EMAILHOST) {
+      return 'Please configure your .env file properly';
+    }
+
+    return nodemailer.createTransport({
+      host: EMAILHOST,
+      port: EMAILPORT || 2525,
+      auth: {
+        user: EMAILUSER,
+        pass: EMAILPASS
+      }
+    });
+  }
+
+
+  /**
+   * @Represents sendEmail
+   * @return {string|Object} Success message | Error
+   */
+  async sendEmail() {
+    const { mailOptions } = this.mailOptions;
+
+    try {
+      const mail = await EmailNotificationAPI.transportCreator()
+        .sendMail(mailOptions);
+      if (mail.response.includes('OK')) return ('Message sent');
+    } catch (error) {
+      return (error);
```

Comment:
Why do you have to put the return value in parentheses?

Reply:
I intended returning more than 1 variables before, but its no longer needed now.